addon/services/aviary-api-client.js

```diff
@@ -5,19 +5,22 @@ import Ember from 'ember';
 export default Ember.Object.extend({
   featherEditor: null,
 
-  init: function() {
+  launch: function(options, initOptions) {
     var featherEditor = this.get('featherEditor');
     if (!featherEditor) {
-      var aviaryFeather = new Aviary.Feather({
-        apiKey: this.get('apiKey'),
-        apiVersion: 3,
-        theme: this.get('theme')
-      });
-      this.set('featherEditor', aviaryFeather);
+      this.initFeatherEditor(initOptions);
```

Comment:
This will lazy load feather editor. We didn't want this behavior because a user will try to launch the editor and get no response for the time it takes for aviary to respond.